.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [11, 14]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Build with Maven
+        run: mvn -B verify
+      - name: Integration Tests
+        run: mvn -B --projects integration-tests -PintegrationCI surefire:test
```

Comment:
```
mvn -B --projects integration-tests -Pintegration -Dspring.profiles.active=ci surefire:test
```

War Original-Zeile in alter GitLab-CI 🤔

Reply:
Hoppla, wo kommt dass den plötzlich her? Ist gefixed.